src/Modal/ModalContents.js

```diff
@@ -282,13 +282,11 @@ class ModalContents extends Util.mixin(BindMixin) {
     }
 
     let calculatedHeight = this.heightInfo;
-
     let contentHeight = calculatedHeight.contentHeight;
-    let modalStyle = {height: calculatedHeight.height};
+    let modalStyle = null;
 
-    let useScrollbar = false;
-    if (calculatedHeight.height !== 'auto') {
-      useScrollbar = true;
```

Comment:
I think you still want to use this `if` statement, you just want to allow override from the prop. Like this:
```
let useScrollbar = false;
if (this.props.useGemini && calculatedHeight.height !== 'auto') {
  useScrollbar = true;
}
```